starfish/core/binary_mask/binary_mask.py

```diff
@@ -157,14 +159,15 @@ def from_label_image(
         label_image : int array
             Integer array where each integer corresponds to a region.
         physical_ticks : Dict[Coordinates, Sequence[float]]
-            Physical coordinates for each axis.
+            Physical ticks for each axis.
 
         Returns
         -------
         masks : BinaryMaskCollection
             Masks generated from the label image.
         """
         props = regionprops(label_image)
+        len_max_label = len(str(len(props) - 1))
```

Comment:
why is there the extra str conversion here?

Reply:
I was tripped up by this too.  Let's say we have 100 regions.  len(props) - 1 gets you 99 as an int.  Then str(99) gives you "99".  Then len("99") gives you 2.